Extensions/NotificationService/NotificationService.swift

```diff
@@ -8,23 +8,24 @@ import Shared
 import Storage
 import Sync
 import UserNotifications
-import os.log
-
-func consoleLog(_ msg: String) {
-    os_log("%{public}@", log: OSLog(subsystem: "org.mozilla.firefox", category: "firefoxnotificationservice"), type: OSLogType.debug, msg)
-}
+import Logger
 
 class NotificationService: UNNotificationServiceExtension {
     var display: SyncDataDisplay?
     var profile: ExtensionProfile?
+    let logger: Logger
+
+    init(logger: Logger = DefaultLogger.shared) {
+        self.logger = logger
+    }
 
     // This is run when an APNS notification with `mutable-content` is received.
     // If the app is backgrounded, then the alert notification is displayed.
     // If the app is foregrounded, then the notification.userInfo is passed straight to
     // AppDelegate.application(_:didReceiveRemoteNotification:completionHandler:)
     // Once the notification is tapped, then the same userInfo is passed to the same method in the AppDelegate.
     override func didReceive(_ request: UNNotificationRequest, withContentHandler contentHandler: @escaping (UNNotificationContent) -> Void) {
-        consoleLog("push received")
+        logger.log("push notification received", level: .info, category: .lifecycle)
```

Comment:
Oh, I had not realized we could use .lifecycle as such. Good point. I think in my other PR removing .core we could replace some with that category as well. Let me know what you think.

Reply:
Yeah that could work for some of them. I think we might also want to add a general category too, I feel there will be many cases where people don't know which category to use so will misuse them. Better to have them end up in a default or unlabelled category than to add unwanted noise to the others.

Reply:
Ok I'll add another task for this, good point. `Unlabeled` make sense, as its less enticing to use than `General` (which could refrain people from using another category the log would fit in).